broadinstitute/single_cell_portal_core
# (SCP-3595) Revert collections UX PR

(SCP-3595) This is a straight revert of the collections UX PRs, with the addition of a migration script to fix any environments which already deleted the covid19 feature flag.  

MANUAL TESTING:
 1. Check out branch, run migrations
 2. Confirm home page has covid19 space link
 3. confirm nav menu refers to 'Branding groups'
 4. Go to the Branding Groups admin page.  Click around and confirm everything says 'Branding group' rather than Collection
